test/4_permissions_test.js

```diff
@@ -67,6 +69,20 @@ contract('LinniaPermissions', accounts => {
       assert.equal(await newInstance.hub(), hub.address);
     });
   });
+  describe('delegate', () => {
+    it('should not let non-user set a delegate', async () => {
+      await assertRevert(
+        instance.addDelegate(delegate, {from: nonUser})
+      );
+    });
+    it('should set a delegate', async () => {
+      const tx = await instance.addDelegate(delegate, {from: user1});
+      assert.equal(await instance.delegates.call(user1, delegate), true);
```

Comment:
switch the order of these lines.

Reply:
can you explain your reasoning behind this please? we set the delegate in 79 and then test it in 80.

Reply:
can you move line 80 to line 84

Reply:
cool